src/utils/CourseChecklist/courseChecklistValidators.js

```diff
@@ -3,7 +3,7 @@ export const hasWelcomeMessage = updates => (
 );
 
 export const hasGradingPolicy = grades => (
-  grades.sum_of_weights === 1
+  parseFloat(grades.sum_of_weights.toPrecision(2), 10) === 1.0
```

Comment:
This address floating point number approximation. There was an instance of a course that had a grading policy of something like `1.00000000004`, which was failing the check.